src/main/java/net/hypixel/nerdbot/generator/impl/tooltip/MinecraftTooltipGenerator.java

```diff
@@ -105,7 +105,12 @@ public MinecraftTooltip parseLore(String input, TooltipSettings settings) {
             if (settings.isEmptyLine()) {
                 builder.withEmptyLine();
             }
-            builder.withLines(LineSegment.fromLegacy(rarity.getFormattedDisplay() + " " + settings.getType(), '&'));
+
+            builder.withLines(LineSegment.fromLegacy(
+                rarity.getFormattedDisplay() +
+                    (settings.getType() == null || settings.getType().isEmpty() ? "" : " " + settings.getType()),
```

Comment:
We should assign this to a variable beforehand and reference the variable, keeping the method in one line